docs/source/pillars/computing_technology.md

```diff
@@ -112,6 +112,15 @@ _The ability to run analyses requiring more compute resources, or more specialis
     For example, when using physical compute resources all sensitive data must be securely wiped before another user is given access to that same node.
     In a cloud hosted TRE virtual machines should be destroyed and recreated.
   - Mandatory
+* - TREs working with big datasets could integrate with large-scale data analytics tools.
+  - For example Spark and Hadoop can be used for distributed computing across a cluster.
+    Where data included in TREs are impractically large for single-machine computing, this will be an advantage.
+  - Optional
+* - TREs integrating with cloud-native managed services should consider the associated risks.
+  - Public clouds provide many different managed services which might include Jupyter notebooks, big data pipelines, databases, API management or serverless computing.
+    Examples include Databricks for automated cluster management and cloud machine-learning platforms such as SageMaker (AWS) or Azure Machine Learning.
```

Comment:
```suggestion
  - Cloud providers supply many different managed services.
```

Private cloud providers also.

I think listing types of services and (especially) what they do is redundant here. I would want to remove that to keep the meaning of this section clear.